hack/lib/golang.sh

```diff
@@ -481,7 +481,7 @@ EOF
   IFS=" " read -ra go_version <<< "$(GOFLAGS='' go version)"
   local minimum_go_version
   # TODO(liggitt): Need to switch this to 1.19 once we update images to newer go version
-  minimum_go_version=go1.17.0
+  minimum_go_version=go1.19
```

Comment:
revert this change, per https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3744-stay-on-supported-go-versions#proposal we'll continue to run unit and integration tests on the original go version for 1.23, which requires leaving the minimum go version in golang.sh as 1.17.x

can drop the TODO or change the comment to explain why this is 1.17

Reply:
done